kubernetes/gaffer/values.yaml

```diff
@@ -44,7 +45,13 @@ ui:
     }
 
   replicaCount: 1
-  resources: {}
+  resources:
+    requests:
+      memory: "400Mi"
+      cpu: "100m"
+    limits:
+      memory: "400Mi"
+      cpu: "100m"
```

Comment:
Best practice is not to set any default resource requests/limits as it prevents deployment to resource constrained environments.